next-tavla/src/Admin/scenarios/BoardTitle/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { SecondarySquareButton } from '@entur/button'
+import { TextField } from '@entur/form'
+import { CheckIcon, CloseIcon, EditIcon } from '@entur/icons'
+import { Heading1 } from '@entur/typography'
+import { useSettingsDispatch } from 'Admin/utils/contexts'
+import { useState } from 'react'
+import classes from './styles.module.css'
+
+function BoardTitle({ title }: { title: string }) {
+    const [isEditing, setIsEditing] = useState<boolean>(false)
+    const dispatch = useSettingsDispatch()
+
+    let newTitle = title
```

Comment:
Syntes vi kan droppe newTitle, lag en hjelpefunksjon for å dispatche endringer i settings, og bruk denne direkte alle stedene du bruker newTitle. Jeg antar at dette vil gå på bekostning av muligheten til å reversere/avbryte en tittelendring, men det er helt ok.

Reply:
Diskuterte dette litt med Natasha før hun overtar nå, og å scrappe muligheten for å kunne avbryte endringer føler vi er en design-avgjørelse som ikke stemmer helt med det som er gjort tidligere, og vi ser ikke helt den tekniske begrunnelsen. Kunne vi tatt en diskusjon på dette litt senere?

![bitmoji](https://sdk.bitmoji.com/render/panel/d8e66e64-ec9c-495f-947e-9bfe52d2523f-2070b7f9-1921-49b1-ae56-267ad10962be-v1.png?transparent=1&palette=1&width=246)

Reply:
Jeg kan være enig i det, skriv det om til å bruke en state variabel for midlertidig tittel så blir det nok mer logisk